samcli/lib/hook/hook_config.py

```diff
@@ -87,3 +87,7 @@ def description(self) -> Optional[str]:
     @property
     def functionalities(self) -> Dict[str, HookFunctionality]:
         return cast(Dict[str, HookFunctionality], self._config["functionalities"])
+
+    @property
+    def iac_framework(self) -> str:
+        return cast(str, self._config.get("iac_framework") if self._config.get("iac_framework") else "")
```

Comment:
Would this be the same?
```suggestion
        return cast(str, self._config.get("iac_framework", ""))
```